### PR TITLE
MOTECH-2759: Added integration test for create patient post action parameter.

### PR DESCRIPTION
**Pull request by afijal**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by kmadej** [comment]
test this please

File: openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java [hunk 1]
```diff
@@ -210,6 +220,39 @@ public void testOpenMRSProviderDataSourceAndCreateEncounterAction() throws Inter
         checkIfProviderWasCreatedProperly();
     }
 
+    @Test
+    public void testCreatePatientPostActionParameter() throws InterruptedException {
```
Comment (kmadej) [inline openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java:224]: 1. this task logic should be in separate method
2. please use english example names : John Smith and so on

File: openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java [hunk 2]
```diff
@@ -166,6 +166,16 @@ public void tearDown() throws PatientNotFoundException {
             patientService.deletePatient(DEFAULT_CONFIG_NAME, patient.getUuid());
         }
 
+        patient = patientService.getPatientByMotechId(DEFAULT_CONFIG_NAME, "Robert Kowalski");
```
Comment (kmadej) [inline openmrs/src/test/java/org/motechproject/openmrs/it/MRSTasksIntegrationBundleIT.java]: change Robert Kowalski and Jan Nowak

**Comment by kmadej** [comment]
test this please

**Pull request merged**